src/main/java/com/starbucks/model/Status.java

```diff
@@ -0,0 +1,5 @@
+package com.starbucks.model;
+
+public enum Status {
+    ORDERED, INPROCESS, READY
```

Comment:
Better to use IN_PROGRESS instead of INPROCESS, not critical but it's why we need a code review so another person would point on things that catch their eye and annoy them )